common/client_flags_test.go

```diff
@@ -110,5 +110,18 @@ var _ = Describe("client flags", func() {
 				Password: expectedPassword,
 			},
 		}),
+		Entry("proxy auth", []string{
+			"--proxy-auth",
+		}, &ClientConfig{
+			Rode: &RodeClientConfig{
+				Host: "rode:50051",
+			},
+			OIDCAuth: &OIDCAuthConfig{},
+			BasicAuth: &BasicAuthConfig{
```

Comment:
This was copypasta from the other test and it should be failing since those flags aren't being passed here. I think the setup may not be initializing some of these variables.

Reply:
I moved the assignments from the `BeforeEach` into the var declaration; I think all of those assignments were unused because we build the flags before the `BeforeEach` even runs